recipes/6s/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sixs" %}
+{% set version = "1.1" %}
+{% set sha256 = "4253664bfed35a659ac602d4472780df7c665d67021d495327f47eb7325a2d7e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://bitbucket.org/petebunting/6s/get/Version%201.1.tar.gz
```

Comment:
Should we change this to download from http://6s.ltdri.org/pages/downloads.html ?